### PR TITLE
Set up back-end

### PR DESCRIPTION
**Pull request by taylordu**
Created file structure for back-end and created/connected ElephantSQL database with test data

### CODE REVIEW

File: controller/data.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const db = require('../models/database');
```
Comment (kadirgund) [inline controller/data.js:1]: Filename should be controllers.

**Review by CaliskanBurak — APPROVED** [review]
Approved

**Review by kadirgund — APPROVED** [review]
Approved.

**Review by patrick934 — APPROVED** [review]
Approved

**Pull request merged**